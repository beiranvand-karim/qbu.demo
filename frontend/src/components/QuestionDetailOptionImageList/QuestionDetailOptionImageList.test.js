/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {QuestionDetailOptionImageList} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<QuestionDetailOptionImageList />', () => {
   const options = ["option 1", "option 2", "option 3", "option 4"];
   it('should render without crashing', () => {
      shallow(<QuestionDetailOptionImageList options={options} />)
   });
   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><QuestionDetailOptionImageList
         options={options}/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});