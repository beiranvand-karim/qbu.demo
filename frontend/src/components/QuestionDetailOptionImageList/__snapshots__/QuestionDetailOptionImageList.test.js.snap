// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionDetailOptionImageList /> should matches the snapshot 1`] = `
Array [
  <div
    className="position-relative form-check"
  >
    <label
      className="form-check-label"
    >
      <label
        className=""
      >
        <input
          className="form-check-input"
          name="questionDetailRadio"
          onChange={[Function]}
          type="radio"
        />
        <img
          alt="option 1"
          src="option 1"
        />
      </label>
    </label>
  </div>,
  <div
    className="position-relative form-check"
  >
    <label
      className="form-check-label"
    >
      <label
        className=""
      >
        <input
          className="form-check-input"
          name="questionDetailRadio"
          onChange={[Function]}
          type="radio"
        />
        <img
          alt="option 2"
          src="option 2"
        />
      </label>
    </label>
  </div>,
  <div
    className="position-relative form-check"
  >
    <label
      className="form-check-label"
    >
      <label
        className=""
      >
        <input
          className="form-check-input"
          name="questionDetailRadio"
          onChange={[Function]}
          type="radio"
        />
        <img
          alt="option 3"
          src="option 3"
        />
      </label>
    </label>
  </div>,
  <div
    className="position-relative form-check"
  >
    <label
      className="form-check-label"
    >
      <label
        className=""
      >
        <input
          className="form-check-input"
          name="questionDetailRadio"
          onChange={[Function]}
          type="radio"
        />
        <img
          alt="option 4"
          src="option 4"
        />
      </label>
    </label>
  </div>,
]
`;
